azure-communication-ui/calling/src/main/res/values/azure_communication_ui_calling_strings.xml

```diff
@@ -47,6 +47,7 @@
     <string name="azure_communication_ui_calling_view_info_header_waiting_for_others_to_join">Waiting for others to join</string>
     <string name="azure_communication_ui_calling_view_info_header_call_with_1_person">Call with 1 person</string>
     <string name="azure_communication_ui_calling_view_info_header_call_with_n_people">Call with %1$d people</string>
+    <string name="azure_communication_ui_calling_view_info_header_call_timer_format" translatable="false">%02d</string>
```

Comment:
This is a bit of Nit:

But flags are supported in XML as well, i.e.
<!-- <CUSTOM_CALL_HEADER> -->
<!-- </CUSTOM_CALL_HEADER> -->

It's good to wrap it all, because you can use UFM to generate AI summaries and code reviews, and if it sees it all, it'll be more robust. Flagging will ensure in disabled state doesn't leave unused resources in build.